RELEASE_NOTES.md

```diff
@@ -119,3 +119,5 @@ This release replaces the `@actor` decorator with a new `Actor` class.
 - Fix `consumer_power` and `producer_power` similar to `pv_power`
 
 - Zero value requests received by the `PowerDistributingActor` will now always be accepted, even when there are non-zero exclusion bounds.
+
+- Hold on to a reference to all tasks, so they don't get garbage collected.
```

Comment:
Optionally you might want to be more specific about which tasks by mentioning `all streaming tasks in the microgrid API client` or something around that

Reply:
fixed